examples/generative/diffusion/training_loop.py

```diff
@@ -410,6 +426,8 @@ def training_loop(
 
         # Save network snapshot.
         if (snapshot_ticks is not None) and (done or cur_tick % snapshot_ticks == 0):
+            if dataset == "dfsr":
+                print("Saving network snapshot.")
```

Comment:
Same here

Reply:
The print function has been replaced with logger0.info.